the-historical-journal.csl

```diff
@@ -77,7 +94,6 @@
   </macro>
   <macro name="publisher">
     <group delimiter=": ">
-      <text variable="publisher"/>
       <text variable="publisher-place"/>
```

Comment:
@ntdmiller, was this an intentional change? It's a bit uncommon to print the "publisher-place" but not the "publisher" itself.

Reply:
Yes, that's right:
M. Cowling, 1867: Disraeli, Gladstone and revolution: the passing of the
second Reform Bill (Cambridge, 1967), pp. 41-5, 140-7.  
and other examples.